### ISSUE TITLE
Update to the new PowerBI Rest API V2

### ISSUE THREAD

**Issue by diego-d5000** [Microsoft/PowerBI-Node#40]
Hi, are you planning to update this "node powerbi sdk" to use the new powerbi api like in C# client lib https://github.com/Microsoft/PowerBI-CSharp/tree/master/sdk/PowerBI.Api/Source/V2 ?

**Comment by whitneyland** [comment]
@ali-hamud @itsananderson - Questions related to this issue:

How come there has been no response in over a month?
Who owns replying to github issues on this repro, is it you guys, or someone else at MS?
Are you aware that this hurts Microsoft strategically?  

When a company chooses GitHub to help open and establish a new platform like Power BI, responsiveness is what boosts developer momentum, unblocks value added solutions, and spreads success of the platform in general.

If a question pops up here asking "what's is an api?", or something that can be googled in 60 seconds, I agree it shouldn't have some kind of SLA response time.

However in this case it's not a silly question.  In fact it's question fundamental to the road map / direction of the platform / supporting libraries.  People make big decisions on these kind of questions, yet this sits here over a month unanswered.

I know you guys personally are busy.  Maybe that's an understatement, it's pretty easy to get underwater being a dev at MS.  However this is a bigger picture thing.  I'm not sure what the internal process is on this, it just doesn't look good from the outside.

**Comment by itsananderson** [comment]
I'm sorry you haven't gotten a response on this. I am no longer working at
Microsoft, so I'm not in a position to review or accept this contribution.

**Comment by ali-hamud** [comment]
Hi,
sorry it waited that long. it is a long review. it just got less priority relevant to other issues. 
I will update on this next Monday.

**Comment by diego-d5000** [comment]
By the way, if someone want to use updated api/sdk you can point the dependency to my [PR repo](https://github.com/diego-d5000/PowerBI-Node/tree/apiv2) like this:

```
npm install https://github.com/diego-d5000/PowerBI-Node/tarball/apiv2
```

And then if it is accepted you just need to change the url to npm dependency in your package.json

**Comment by whitneyland** [comment]
@ali-hamud hello, I hope you find yourself well.  Based on your comment it was expected this issue would be updated 8/7/2017 (last monday).

Will you please provided a new date as to when we might expect to see this?

**Comment by ali-hamud** [comment]
Hi, as we work on more important features, we decided not to update NodeJS sdk.
in order to prioritize this SDK higher, please vote in powerbi ideas:
https://ideas.powerbi.com/forums/265200-power-bi-ideas/suggestions/20566726-power-bi-nodejs-sdk

Thanks.

**Comment by whitneyland** [comment]
@ali-hamud,

Priorities is not the issue, it's responsiveness/communication.

Im not suggesting Microsoft change the priority of anything.

I'm suggesting that when you publicly promise to provide an update on a certain day, and then don't do it, it hurts the credibility you/MS/the PBI effort.

I'm suggesting that it's fine to deprioritize, pause, or deprecate projects, but that if a GitHub repo exists the status should be clear.

Likewise, it's fine to not accept any PR or fix any given issue, but then they should be quickly closed or marked won't fix.

**Comment by ali-hamud** [comment]
Hi @whitneyland 
I waited to an internal study about this library usage and need.
anyway, meanwhile we decided to focus on other things.
